.travis.yml

```diff
@@ -195,6 +195,10 @@ install:
     echo "constraints:  sqlite-simple < 0"                       >> cabal.project
     echo "allow-newer:  servant-pagination-2.2.2:servant"        >> cabal.project
     echo "allow-newer:  servant-pagination-2.2.2:servant-server" >> cabal.project
+    echo "allow-newer:  servant-multipart:servant"               >> cabal.project
+    echo "allow-newer:  servant-multipart:servant-server"        >> cabal.project
+    echo "allow-newer:  servant-multipart:servant-client-core"   >> cabal.project
+    echo "allow-newer:  servant-js:servant"                      >> cabal.project
```

Comment:
note to self: remove this again when everything is upgraded!  (also further down in this file and in cabal.project)